<ui version="4.0" >
 <class>QVolume3DViewTestingExtensionBase</class>
 <widget class="QWidget" name="QVolume3DViewTestingExtensionBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>773</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>1</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QToolButton" name="m_axialOrientationButton" >
           <property name="text" >
            <string>Axial</string>
           </property>
           <property name="icon" >
            <iconset resource="../../../main/main.qrc" >:/images/axial.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="checkable" >
            <bool>false</bool>
           </property>
           <property name="checked" >
            <bool>false</bool>
           </property>
           <property name="toolButtonStyle" >
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
           <property name="autoRaise" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_sagitalOrientationButton" >
           <property name="text" >
            <string>Sagital</string>
           </property>
           <property name="icon" >
            <iconset resource="../../../main/main.qrc" >:/images/sagital.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="checkable" >
            <bool>false</bool>
           </property>
           <property name="checked" >
            <bool>false</bool>
           </property>
           <property name="toolButtonStyle" >
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
           <property name="autoRaise" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_coronalOrientationButton" >
           <property name="text" >
            <string>Coronal</string>
           </property>
           <property name="icon" >
            <iconset resource="../../../main/main.qrc" >:/images/coronal.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="checkable" >
            <bool>false</bool>
           </property>
           <property name="checked" >
            <bool>false</bool>
           </property>
           <property name="toolButtonStyle" >
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
           <property name="autoRaise" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Minimum</enum>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>5</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Orientation</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QToolButton" name="m_zoomToolButton" >
           <property name="text" >
            <string>Zoom</string>
           </property>
           <property name="icon" >
            <iconset resource="../../../main/main.qrc" >:/images/zoom.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="checkable" >
            <bool>true</bool>
           </property>
           <property name="checked" >
            <bool>false</bool>
           </property>
           <property name="toolButtonStyle" >
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
           <property name="autoRaise" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_rotate3DToolButton" >
           <property name="text" >
            <string>Rotate</string>
           </property>
           <property name="icon" >
            <iconset resource="../../../main/main.qrc" >:/images/rotate3d.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="checkable" >
            <bool>true</bool>
           </property>
           <property name="checked" >
            <bool>true</bool>
           </property>
           <property name="toolButtonStyle" >
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
           <property name="autoRaise" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_panToolButton" >
           <property name="text" >
            <string>Pan</string>
           </property>
           <property name="icon" >
            <iconset resource="../../../main/main.qrc" >:/images/move.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="checkable" >
            <bool>true</bool>
           </property>
           <property name="checked" >
            <bool>false</bool>
           </property>
           <property name="toolButtonStyle" >
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
           <property name="autoRaise" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Minimum</enum>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_5" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>5</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Mouse Function</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <layout class="QHBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="label" >
             <property name="text" >
              <string>Presets</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="m_clutPresetsComboBox" />
           </item>
          </layout>
         </item>
         <item>
          <widget class="QToolButton" name="m_clutEditToolButton" >
           <property name="text" >
            <string>Edit CLUT</string>
           </property>
           <property name="icon" >
            <iconset resource="../../../main/main.qrc" >:/images/clutEdit.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolButtonStyle" >
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
           <property name="autoRaise" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Minimum</enum>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_3" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>5</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>CLUT</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QComboBox" name="m_renderingMethodComboBox" >
         <item>
          <property name="text" >
           <string>Ray Casting</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Ray Casting with Shading</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>MIP</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Texture 3D</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Texture 2D</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Iso Surface</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Contouring</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QWidget" native="1" name="m_shadingOptionsWidget" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>5</hsizetype>
           <vsizetype>4</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QCheckBox" name="m_specularCheckBox" >
            <property name="text" >
             <string>Specular</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="m_specularPowerLabel" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="text" >
             <string>Specular power</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="m_specularPowerDoubleSpinBox" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="decimals" >
             <number>1</number>
            </property>
            <property name="maximum" >
             <double>128.000000000000000</double>
            </property>
            <property name="value" >
             <double>64.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Minimum</enum>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_7" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>5</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Rendering Method</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="udg::Q3DViewer" native="1" name="m_3DView" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>7</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>udg::Q3DViewer</class>
   <extends>QWidget</extends>
   <header>q3dviewer.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../main/main.qrc" />
 </resources>
 <connections/>
</ui>
